docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -25,9 +25,17 @@ The following options can optionally be configured:
 
 {{< schema root="BuildpackArtifact" >}}
 
+**Builder**
+
 `builder` is *required* and tells Skaffold which
 [Builder](https://buildpacks.io/docs/app-developer-guide/build-an-app/) to use.
 
+**User defined environment variables**
+
+`env` makes it possible to pass arbitrary environment variables to the buildpacks.
+This is in general used to adjust the detection and the build phases. For eg., most
+builders let you change runtime versions with environment variables.
```

Comment:
I think it’s buildpacks, not builders, that use the variables to restrict runtimes versions.

Wdyt: _Many buildpacks use environment variables to select specific versions of language runtimes._

Reply:
Thanks. I'll take your text